resModal

pimber opened the pull request:
- Have added a cross so when you have selected a customer to a reservation you have chance to delete it again

pimber committed:
resModal

- Have added a cross so when you have selected a customer to a reservation you have chance to delete it again

pimber committed:
XMerge branch 'main' into resModal

jonasKjellerup commented on Flea/Shared/ReservationForm.razor:
```diff
@@ -94,11 +100,19 @@
 
+    public void ResetSelectedCustomer()
+    {
+        _selectedCustomer = null!;
```
`_selectedCustomer` is nullable. Don't use `null!` for nullable types use `null` instead.

jonasKjellerup commented on Flea/Shared/ReservationForm.razor:
```diff
@@ -94,11 +100,19 @@
+    public void ResetSelectedCustomer()
+    {
+        _selectedCustomer = null!;
+        Reservation.ReservationOwner = null!;
```
You don't need to set this since it doesn't affect rendering and a submissions should be blocked when `_selectedCustomer is null` anyways (of course only when it is a new reservation).

(There is no sound reasoning for using `null!` here)

pimber committed:
Changed null! to null for _selectedCustomer in Reservationform

pimber committed:
Stil fixing an error

pimber committed:
Search funktion resModal

- Hvis ingen lopper matcher search kritere vises der ingen lopper
- Fixed så CSS virker i safari

pimber committed:
Ændret lidt CSS

jonasKjellerup committed:
Removed redundant assignment.

pimber committed:
ReservationForm.razor

- Linje 116 skal være der før at man kan fjerne den valgte kunde ordenligt

jonasKjellerup committed:
Added check to verify that a customer has been selected.

 - Also added styles to indicate that the customer field
   is correctly filled out.

jonasKjellerup committed:
Merge branch 'resModal' of github.com:P3-DAT3-03/Flea into resModal

jonasKjellerup committed:
Undid change from commit a662812

pimber committed:
Større arbejde med at andre CSS

pimber committed:
Merge branch 'resModal' of github.com:P3-DAT3-03/Flea into resModal

jonasKjellerup committed:
Removed duplicate ValidationSummary.

pimber committed:
...

pimber committed:
Merge branch 'resModal' of github.com:P3-DAT3-03/Flea into resModal

pimber committed:
Have changed

- CSS to reservationoverview
- Still needs to have changed CSS in customeroverview and
- And check in kort also needs to be maked

pimber committed:
Opdatering af CSS design

- Har lavet forhåbenligt bedre CSS og design til Booking page

pimber committed:
Customer page

- Har tilpasset CSS til customerpage's så det passer

pimber committed:
Knap til tilføj loppemarkedet også opdateret

LowkeyCoding committed:
Merge branch 'main' into resModal

LowkeyCoding commented on Flea/Shared/CustomerForm.razor.css:
```diff
@@ -1,22 +1,43 @@
+/*    display: inline-block;*/
+/*    margin-left: 10px;*/
+/*}*/
+
```
Er der en grund til at det gamle css stadig er her?

LowkeyCoding commented on Flea/Shared/CustomerForm.razor.css:
```diff
@@ -1,22 +1,43 @@
 
-#move-down {
-    margin-top: 40px;
+#display-inline {
```
Id til styling er no no very bad

jonasKjellerup replied:
I det her tilfælde giver det mere mening bare at lave det som en inline style da det kun er en property. Da det kun skal bruges et sted giver det ikke mening at bruge en class.

Under alle omstændigheder så må et id aldrig navngives efter, hvilken stilistisk funktion den skal udføre. Det skal navngives efter hvad elementet er.

pimber committed:
Market page is "done"

pimber committed:
Remove some code

LowkeyCoding committed:
Merge branch 'main' into resModal

jonasKjellerup committed:
CSS touch ups

jonasKjellerup committed:
Added modal padding and backdrop
